src/btree/bt_debug.c

```diff
@@ -788,15 +804,17 @@ __debug_dsk_col_fix(WT_DBG *ds, const WT_PAGE_HEADER *dsk)
  *     Dump a disk page in debugging mode.
  */
 int
-__wt_debug_disk(
-  WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, const char *ofile, bool dump_app_data)
+__wt_debug_disk(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, const char *ofile,
+  bool dump_all_data, bool dump_key_data, bool dump_value_data)
 {
     WT_DBG *ds, _ds;
     WT_DECL_RET;
     uint32_t flags;
 
     ds = &_ds;
-    flags = dump_app_data ? WT_DEBUG_UNREDACT : 0;
+    flags = dump_all_data ? WT_DEBUG_UNREDACT_ALL : 0;
+    flags |= dump_key_data ? WT_DEBUG_UNREDACT_KEYS : 0;
+    flags |= dump_value_data ? WT_DEBUG_UNREDACT_VALUES : 0;
     WT_RET(__debug_config(session, ds, ofile, flags));
```

Comment:
An assertion here would be useful - 
```
WT_ASSERT(session, !(dump_all_data && (dump_key_data || dump_value_data));
flags = dump_all_data ? WT_DEBUG_UNREDACT_ALL : 0;
flags |= dump_key_data ? WT_DEBUG_UNREDACT_KEYS : 0;
flags |= dump_value_data ? WT_DEBUG_UNREDACT_VALUES : 0;
```

Reply:
I don't understand the distinction "all" as opposed to "keys" + "values".
That is it looks to me like:
`WT_DEBUG_UNREDACT_ALL = WT_DEBUG_UNREDACT_KEYS | WT_DEBUG_UNREDACT_VALUES;`
So my question is it necessary to have the third value? Or does `UNREDACT_ALL` print additional information beyond key + value?

Reply:
As discussed in person, it was decided that the unredact values option is not useful, and hence will be removed.